src/test/java/core/basesyntax/service/FileWriterTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterImpl;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterTest {
+    private static final String TEST_PATH = "src/main/resources/dayreport.csv";
+    private static final String FILE_HEADER = "fruit,quantity";
+    private static List<String> testList = new ArrayList<>();
+    private static FileWriter fileWriter;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        testList.add(FILE_HEADER);
+        testList.add(System.lineSeparator());
+        testList.add("banana,30");
+        testList.add(System.lineSeparator());
+        testList.add("pineapple,22");
+        testList.add(System.lineSeparator());
+        testList.add("apple,50");
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void writeToFileFileExist_OK() {
```

Comment:
```suggestion
    public void writeToFile_validFile_OK() {
```

fix naming in all places